src/utils/util/resolver-pipeline/oscal-profile-resolve-merge.xsl

```diff
@@ -68,6 +90,7 @@
             <xsl:for-each select="$merged-selections/selection">
                 <xsl:sequence select="* except back-matter"/>
             </xsl:for-each>
+            <xsl:apply-templates select="processing-instruction('message-handler')"/>
```

Comment:
Item 3 in PR description also required some handling of the processing instructions so they didn't get lost partway through the pipeline.